api/tests/routes/pro/get_offerer_test.py

```diff
@@ -303,8 +392,8 @@ def test_client_can_know_which_venues_have_non_free_offers_without_bank_accounts
 
         # When
         http_client = client.with_session_auth(pro_user.email)
-
-        response = http_client.get(f"/offerers/{offerer.id}")
+        with testing.assert_num_queries(11):
+            response = http_client.get(f"/offerers/{offerer.id}")
```

Comment:
Tu as probablement une requête comptée pour l'accès au `.id` de `offerer` dans ce test ; il est judicieux de le sortir du bloc : 
```suggestion
        offerer_id = offerer.id
        with testing.assert_num_queries(10):
            response = http_client.get(f"/offerers/{offerer_id}")
```

Reply:
yes en effet, j’enlève cf comment ci-dessus